core/arch/arm/pta/system.c

```diff
@@ -40,6 +52,66 @@ static TEE_Result system_rng_reseed(struct tee_ta_session *s __unused,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result system_derive_ta_unique_key(
+			struct tee_ta_session *s,
```

Comment:
Please align with the `(`

Reply:
Sure, will fix.

Reply:
could shorten the function label to `derive_ta_unique_key()`, this is a local function.